### ISSUE TITLE
Allow read-only paths

### ISSUE THREAD

**Issue by djl** [lidarr/Lidarr#815]
**Is your feature request related to a problem? Please describe.**
Lidarr doesn't allow read-only paths.

**Describe the solution you'd like**
A way to add read-only paths.

**Describe alternatives you've considered**
N/A

**Additional context**

My use of Lidarr (and Headphones, which I'm looking to replace) doesn't require it to have write access to my music library.

My current setup works like this:

1. I download music
2. I use [beets](https://github.com/beetbox/beets) to import the music into my library
3. A custom beets plugin adds any new artists to Headphones (or Lidarr, hopefully!)
4. Headphones/Lidarr starts tracking new artists

At no point does Lidarr require write access to my library - it's effectively just a way to track upcoming releases for artists already in my library.

Would it be possible to add support for read-only paths?

**Comment by pagdot** [comment]
I have the same issue. I use beets to manage tracks in my library and have the download directory separated from my library. I'd still like for lidarr to keep track of my library.

**Comment by bakerboy448** [comment]
No Plans at this time.

**Comment by Kernald** [comment]
Are there any plans to ever revisit this in the future? I have the exact same use-case as above, and reading threads like [this](https://www.reddit.com/r/Lidarr/comments/rn0e3r/lidarr_just_nuked_a_compilation_album_from_2005/) is another reason why I really don't want to give such a software write access to my collection. Having a way to monitor it for new artists and existing albums only would be really nice.

**Comment by marcw** [comment]
When using Docker, I wonder if this could be done with:
- mounting the collection directory as a read-only volume
- disabling the "completed download handling" setting in Lidarr
- disabling any track renaming option/file management setting in Lidarr

**Comment by trulow** [comment]
I agree with others. It would be nice to be able to import a "root" directory that's read only. I manage my library externally so I have no need for Lidarr to manage any of my existing collections but it would be nice to import so Lidarr knows what I have versus what I don't have.

**Comment by pagdot** [comment]
I'll try to do it by using overlayfs and mounting the import folder as `rw` over the `ro` music folder

**Comment by altosys** [comment]
With docker this is entirely possible by mounting the music downloads and/or library with `:ro`, it works really well with for instance beets where you can manage your library with it and lidarr will regularly scan for updates from disk and display the albums found in the library.

**Comment by pagdot** [comment]
Oh, I thought that there were issues with it being unable to write to it
